docs/hub/debug.md

```diff
@@ -0,0 +1,32 @@
+# Debug the Mercure.rocks Hub
+
+The hub is shipped with [`pprof`](https://blog.golang.org/pprof),
+a profiler allowing to find bottlenecks, memory leaks and blocked goroutines 
+among other things.
+
+To enable the profiler, add the `debug` global directive to your `Caddyfile`:
+
+```Caddyfile
+{
+    debug
```

Comment:
You must add `admin` not `debug`, that worked for me and caddy docs tells:
> debug enables debug mode, which sets all log levels to debug (unless otherwise specified).

And if you want it to work with docker you must enabled it like this `admin 0.0.0.0:2019`.